CoreUtilities/src/au/gov/asd/tac/constellation/utilities/threadpool/UniversalThreadPool.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.threadpool;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * This class holds global objects for the variouse thead pools needed by the
+ * application
+ *
+ * @author altair1673
+ */
+public class UniversalThreadPool {
```

Comment:
Can we rename this to something like `ConstellationGlobalThreadPool`? Universal doesn't seem quite like the correct wording here (as it really is only for Constellation code use)